doc/sphinx/tutorial.md

```diff
@@ -3,86 +3,94 @@ discourse: 4911,5164,5603,6756,8037
 ---
 
 # Tutorial
+This tutorial will instruct you on how to get started building your first
+Wayland compositor with Mir. Additionally, we will provide information on how
+to install some demo compositors that will give you a sense of what you can
+build with Mir.
 
-Mir is a library for building things, not an end-user product, but it does come
-with some demos to illustrate the possibilities.
-
-## Getting Mir demos
-
+## Installing Mir Libraries
 The Mir libraries and demos are available on Ubuntu, Fedora and Arch. It has
-also been built and tested on Debian but, at the time or writing, is not in the
-archive.
-
-For Linux distributions that don't currently package Mir you need to build it
-yourself. (See [Getting Involved in Mir](how-to/getting_involved_in_mir)).
-
-## Getting Mir on Ubuntu
-
-You can install the Mir examples along with the Mir graphics drivers as follows:
-
-    sudo apt install mir-demos mir-graphics-drivers-desktop
-
-It is also useful to install Wayland support for Qt and qterminal:
-
-    sudo apt install qterminal qtwayland5
-
-### Getting the latest Mir release on Ubuntu
+also been built and tested on Debian but it is not in the archive.
```

Comment:
I know this is existing, but [Mir's in Debian](https://tracker.debian.org/pkg/mir).